nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -431,36 +441,48 @@ def output_filter_func(point):
 
     def get_statistic_points(self, model: TModel) -> StatisticPointsContainer:
         self._set_backend_entity(model)
-        nncf_graph = NNCFGraphFactory.create(model) if self.nncf_graph is None else self.nncf_graph
+        model_copy = self._remove_fq_from_inputs(deepcopy(model))
```

Comment:
Should we use this method here?
https://github.com/openvinotoolkit/nncf/blob/7a1de11e3b9c9d641b7f44faa271ac059700f252/nncf/common/utils/backend.py#L90-L101
@l-bat This issue was fixed?

Reply:
The issue was fixed om master branch. We should wait for release of `openvino-dev`.